### PR TITLE
Add usage example to math.poly_val

### PR DESCRIPTION
**Pull request by RichardXiao13**

### CODE REVIEW

File: tensorflow/python/ops/math_ops.py [hunk 1]
```diff
@@ -4228,6 +4228,11 @@ def polyval(coeffs, x, name=None):
 
      p(x) = coeffs[n-1] + x * (coeffs[n-2] + ... + x * (coeffs[1] +
             x * coeffs[0]))
+            
+Usage Example:
+  >>> coefficients = [1.0, 2.5, -4.2]
+  >>> x = 5.0
+  >>> y = tf.math.polyval(coefficients, x)
```
Comment (HotPotatoC) [inline tensorflow/python/ops/math_ops.py]: You should indent the 'Usage example' and the code on the same level.

File: tensorflow/python/ops/math_ops.py [hunk 2]
```diff
@@ -4229,7 +4229,7 @@ def polyval(coeffs, x, name=None):
      p(x) = coeffs[n-1] + x * (coeffs[n-2] + ... + x * (coeffs[1] +
             x * coeffs[0]))
             
-Usage Example:
+  Usage Example:
   >>> coefficients = [1.0, 2.5, -4.2]
   >>> x = 5.0
   >>> y = tf.math.polyval(coefficients, x)
```
Comment (HotPotatoC) [inline tensorflow/python/ops/math_ops.py]: You should provide the output for the doctest. You can do that by removing `y =` then provide the output below that

Comment (RichardXiao13) [reply tensorflow/python/ops/math_ops.py]: I have provided the output.

File: tensorflow/python/ops/math_ops.py [hunk 3]
```diff
@@ -4228,6 +4228,12 @@ def polyval(coeffs, x, name=None):
 
      p(x) = coeffs[n-1] + x * (coeffs[n-2] + ... + x * (coeffs[1] +
             x * coeffs[0]))
+            
+  Usage Example:
```
Comment (lamberta) [inline tensorflow/python/ops/math_ops.py]: Thanks. Please add newline to separate text and code blocks for everything to render on tensorflow.org

**Review by lamberta — APPROVED** [review]
Thank you

**Pull request merged**